src/Components/Header.jsx

```diff
@@ -17,6 +17,11 @@ function Header() {
          <br />
          <br />
          <br />
+         <br />
+         <br />
+         <br />
```

Comment:
br tags should not be used for styling. Check the web docs for more information.